docs/ImportONNXDefs.md

```diff
@@ -1,46 +1,89 @@
 <!--- SPDX-License-Identifier: Apache-2.0 -->
+# Table of Contents
+  1. [Overview](#overview)
+  2. [Add an Operation](#add_operation)
+  3. [Customize an Operation](#customize)
+  4. [Build](#build)
+  5. [Details about version](#version)
+# Overview <a name="overview"></a>
+Onnx-mlir defines an onnx dialect to represent operations specified by onnx.The onnx dialect is createdd with MLIR table
+gen tool. The definition of each operation is transferred from onnx automatically with a 
+ python script, 
+[utils/gen_onnx_mlir.py](../utils/gen_onnx_mlir.py). 
+This script retrieves operation definition from
+ onnx package to generate ONNXOps.td.inc for dialect table gen and OpBuilderTable.inc for 
+onnx model importer in onnx-mlir. 
+The following sections will describe how to use gen_onnx_mlir.py to add an operation into onnx
+ dialect in onnx-mlir and how to refine the definition of the operation.
 
-# Import ONNX specifications into ONNX-MLIR
+# Add an Operation <a name="add_operation"></a>
+To generate an operation for onnx dialect, add this operation into the dictionary, 
+'version_dict', in gen_onnx_mlir.py. 
+The key of this directory is the operation name and the value is the list of 
+opset for this operation. Usually only the top version opset of this operation (in onnx-mlir/third_party/onnx) is supported. Details about versioning can be found in [version section](#Operation Version).
```

Comment:
link is broken when viewed here https://github.com/chentong319/onnx-mlir/blob/tong-docs/docs/ImportONNXDefs.md. VSCode also has a way to visualize md files